src/main/java/frc/robot/physicalConstants.java

```diff
@@ -0,0 +1,146 @@
+// Copyright 2021-2024 FRC 6328
+// http://github.com/Mechanical-Advantage
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// version 3 as published by the Free Software Foundation or
+// available in the root directory of this project.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+package frc.robot;
+
+import edu.wpi.first.math.util.Units;
+
+public final class physicalConstants {
+  // public static final Mode currentMode = Mode.REAL;
+  // public static final double ROBOT_LOOP_PERIOD_SECS = 0;
+  // public static final double PIVOT_ZERO_ANGLE = 59;
+
+  public static final Mode currentMode = Mode.REAL;
+  public static final boolean tuningMode = true;
+  public static final String CANBUS = "CAN Bus 2";
+  public static final double LOOP_PERIOD_SECS = 0.02;
+
+  public static final String LL_ALIGN = "limelight-align";
```

Comment:
In our project template planning document we discussed addresses like this and the name of the canbus going in robot map